docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -55,6 +55,29 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 <10> In order to release all transient resources (memory, threads, ...) a `CacheManager` provides to `Cache` instances
      it manages, you have to invoke `CacheManager.close()`, which in turns closes all `Cache` instances known at the time.
 
+=== Clustered cache manager
+
+In addition to the steps performed for creating standalone cache manager, clustered cache manager creation requires you to provide the clustering
```

Comment:
I agree with Chris. Phrasing should be something like: _the cache manager with clustering support_